store.go

```diff
@@ -1749,6 +1729,23 @@ func (s *store) Container(id string) (*Container, error) {
 	return rcstore.Get(id)
 }
 
+func (s *store) ContainerLayerID(id string) (string, error) {
+	rcstore, err := s.ContainerStore()
+	if err != nil {
+		return "", err
+	}
+	rcstore.Lock()
+	defer rcstore.Unlock()
+	if modified, err := rcstore.Modified(); modified || err != nil {
+		rcstore.Load()
+	}
+	container, err := rcstore.Get(id)
+	if err != nil {
```

Comment:
Do you have to verify that the container != nil and handle that too?  Or will the return line 1746 be happy if the container is nil?